index.html

```diff
@@ -799,6 +802,42 @@ <h2>
           <li>Let |acquisitionTime:EpochTimeStamp| be a new {{EpochTimeStamp}}
           that represents now.
           </li>
+          <li>
+            <ins cite="#a4"> Check if the emulated position data should be used by running
+              the following steps:
+              <ol>
+                <li>Let |emulated position data| be [=get emulated position data=]
```

Comment:
For readability, please use lowerSnakeCase for variables.

```suggestion
                <li>Let |emulatedPositionData| be [=get emulated position data=]
```